client/received_blocks_map.go

```diff
@@ -0,0 +1,303 @@
+// This file was automatically generated by genny.
+// Any changes will be lost if this file is regenerated.
+// see https://github.com/cheekybits/genny
+
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
few nits:
- automate generation of the genny output files (`m3db/generated/genny/generate.go`, `make genny-gen`, etc)
- filter out the generated files from the coverage report (can include filename patterns in `.excludecoverage`)